rsv-metrics/libexec/probes/gratia-config-probe

```diff
@@ -268,77 +268,92 @@ EOF
 
   ################################
   # Check last contact time with Gratia server.
-  my $wget_cmdline = shellquote(
-    qw(wget -q --tries=1 --timeout=10 -O -),
-    "http://$soaphost/gratia-administration/monitor-status.html?probename=$probename"
+  my @cur_time= localtime();
+  # localtime gives years from 1900
+  my $year = 1900 + $cur_time[5];
+  # perl uses 0 based months
+  my $month = 1 + $cur_time[4];
+  my $gracchost = "https://gracc.opensciencegrid.org";
+  my $graccindex = sprintf "gracc.osg.raw-%04d.%02d", $year, $month;
+  my $curl_cmdline = shellquote(
+    qw(curl -q --retry 1  --connect-timeout 10 -XGET ),
+    "$gracchost/q/$graccindex/_search",
+    "-d",
+    qq({ "query": { "query_string": { "query": "ProbeName:\\"$probename\\" AND EndTime:[now-2d TO now]" } }, "sort": [ { "\@received": { "order": "desc" } } ],"size": 1 })
```

Comment:
I would feel so much better about this if we could use some library for creating & parsing json ... `perl-JSON`, for instance, is available from the OS repos and we could add it to the rpm requirements.  (Either now or else in a future release...)

Reply:
Yeah, I can make the changes to use perl-JSON or similar.  I can get the updates in by tomorrow morning.